docs/install.rst

```diff
@@ -9,292 +9,60 @@ Simple Installation
 %%%%%%%%%%%%%%%%%%%%%%%%
 
 PyCBC is available through the PyPI. For straightforward use of the PyCBC library
-and executables, we recommend installing with the following command. If you 
+and executables, we recommend installing with the following command. If you
 are not running in a specialized computing environment, this is probably the
 appropriate thing to do.
 
 .. code-block:: bash
-    
+
     pip install lalsuite pycbc
```

Comment:
Is this spacing not needed for the formatting in Sphinx? Can you check this still displays?

Reply:
Only change is whitespace removal. The newline is still there.

Reply:
Yes, this displays correctly.